vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -0,0 +1,168 @@
+
+2. Install the Solr and Zookeeper CRDs:
+```
+kubectl create -f https://solr.apache.org/operator/downloads/crds/v0.5.0/all-with-dependencies.yaml
+```
+3. Install the Solr operator and Zookeeper Operator:
+```
+helm install solr-operator apache-solr/solr-operator --version 0.5.0
+```
+4. Create a descriptor file `deployment-solrCloud.yaml` following the sample below. Use the domain name of your Entando application.
+
+For example:
+``` yaml
+apiVersion: solr.apache.org/v1beta1
+kind: SolrCloud
+metadata:
+  name: solr
+spec:
+  solrAddressability:
+    external:
+      domainName: YOUR-DOMAIN
+      method: Ingress
+      useExternalAddress: true
+  customSolrKubeOptions:
+    podOptions:
+      resources:
+        limits:
+          memory: 2Gi
+          cpu: 1000m
+        requests:
+          cpu: 350m
+          memory: 2Gi
+  dataStorage:
+    persistent:
+      pvcTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 10Gi
+      reclaimPolicy: Delete
+  replicas: 3
+  solrImage:
+    repository: entando/entando-solr
+    tag: "8"
+  solrJavaMem: -Xms2048M -Xmx2048M
+  updateStrategy:
+    method: StatefulSet
+  zookeeperRef:
+    provided:
+      chroot: /explore
+      image:
+        pullPolicy: IfNotPresent
+        repository: pravega/zookeeper
+        tag: 0.2.13
+      persistence:
+        reclaimPolicy: Delete
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+      replicas: 3
+      zookeeperPodPolicy:
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 250m
+            memory: 500Mi
+
+```
+
+3. Apply the Solr application YAML:
+```
+kubectl apply -f deployment-solrCloud.yaml -n entando
+```
+4. To check that the service has started properly:
+```
+kubectl get solrclouds -w
+```
+You should see a response similar to this:
+```
+NAME   VERSION   TARGETVERSION   DESIREDNODES   NODES   READYNODES   UPTODATENODES   AGE
+solr   8                         1              1       1            1               79m
+```
+
+## Solr Standalone 
+
+### Create the Core
+1. Shell into the Solr pod:
+``` 
+kubectl exec -it YOUR-SOLR-POD-NAME -- /bin/bash
+```
+2. Generate the core, replacing `entando` with your namespace as needed:
+```
+bin/solr create_core -c entando
+```
+
+### Edit the Entando App Engine Deployment
+1. Scale down the App Engine deployment (entando-de-app image) to 0.
+2. Edit the App Engine deployment and add the following env variables:
+
+```
+spec
+  -env
+    - name: SOLR_ACTIVE
+      value: "true"
+    - name: SOLR_ADDRESS
+      value: http://solr:8983/solr
+```
+3. Scale the `entando-de-app` deployment back up to 1 and check the system.
+
+## Solr Cloud for Multitenancy Application
+
+### Generate the Solr Core
+
+1. Generate the Solr core with the following command where  
```

Comment:
I was not able to do a deep analysis but the ingress I have after having followed the previous steps doesn't match that pattern. I guess there is something missing or not correct in the previous steps that we should fix to make this step works.